engine/src/main/java/org/entando/entando/aps/system/services/widgettype/WidgetService.java

```diff
@@ -234,6 +242,8 @@ public WidgetDto updateWidget(String widgetCode, WidgetRequest widgetRequest) {
                 }
             }
             this.addFragments(widgetDto);
+        } catch (ValidationGenericException vge) {
```

Comment:
Log this. No exceptions without logging.

Reply:
Done!